src/routers/upload.py

```diff
@@ -297,25 +295,76 @@ async def upload_geotiff_chunk(
 				initial_dataset,
 			)
 		)
-		# loop = asyncio.get_running_loop()
-		# loop.run_in_executor(
-		# 	executor,
-		# 	combine_chunks,
-		# 	tmp_dir,
-		# 	chunks_total,
-		# 	file_name,
-		# 	target_path,
-		# 	token,
-		# 	initial_dataset,
-		# )
-
-		# executor.submit(combine_chunks, tmp_dir, chunks_total, file_name, target_path, token, initial_dataset)
 
 		return initial_dataset
 
 	return {'message': f'Chunk {chunk_index} of {chunks_total} received'}
 
 
+@router.put('/datasets/{dataset_id}/label-object')
```

Comment:
If I may be really specific here: I think this is a POST and not a PUT route. The difference is that PUT is *idempotent*, meaning, if you call this route twice, the data is **updated**. This is not the case, if I get this right. If you call the route twice, you end up with two uploaded objects. Thus this route should be a POST.